src/btrfsbalancer.h

```diff
@@ -0,0 +1,83 @@
+/****************************************************************************************
+**
+** Copyright (C) 2015 Jolla Ltd.
+** Contact: Martin Grimme <martin.grimme@gmail.com>
+** All rights reserved.
+**
+** This file is part of the btrfs-balancer package.
+**
+** You may use this file under the terms of the GNU Lesser General
+** Public License version 2.1 as published by the Free Software Foundation
+** and appearing in the file license.lgpl included in the packaging
+** of this file.
+**
+** This library is free software; you can redistribute it and/or
+** modify it under the terms of the GNU Lesser General Public
+** License version 2.1 as published by the Free Software Foundation
+** and appearing in the file license.lgpl included in the packaging
+** of this file.
+**
+** This library is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+** Lesser General Public License for more details.
+**
+****************************************************************************************/
+
+
+#ifndef BTRFSBALANCER_H
+#define BTRFSBALANCER_H
+
+#include "btrfs.h"
+
+#include <QObject>
+#include <QString>
+
+class BtrfsBalancer : public QObject
+{
+    Q_OBJECT
+public:
+    enum Status
+    {
+        READY = 0,
+        BALANCING
```

Comment:
I'm not sure if I can agree with that as general rule. Yea, the diff will be shorter, but then again on most places new enum values are not much expected. Actually on that department, wonder if balancer can be ever expected to be other than balancing or idle, for which a boolean could suffice. But just bikeshedding, ok with anything :)